src/psyclone/tests/nemo/transformations/openmp/openmp_test.py

```diff
@@ -89,6 +89,36 @@ def test_omp_explicit_gen():
     assert expected in gen_code
 
 
+def test_omp_private_declaration():
+    ''' Check code generation and private/shared declaration when
+    an assignment is parallelised. In this case the code is like:
+    !$omp parallel default(shared), private()
+    jpk = 100
+    do k=1, jpk ...
+    enddo
+    !$omp end parallel
+    do k=1, jpk ...
+
+    In this case jpk should not be declared private, since then it
+    is not defined in the next loop.'''
+
+    _, invoke_info = parse(os.path.join(BASE_PATH,
+                                        "explicit_do_two_loops.f90"),
+                           api=API, line_length=False)
+    psy = PSyFactory(API, distributed_memory=False).create(invoke_info)
+    schedule = psy.invokes.get('explicit_do').schedule
+    schedule.view()
```

Comment:
Please remove this line.

Reply:
Done (I removed all of them, there seems to be no sense in the others either).